packages/bruno-app/src/utils/importers/openapi-collection.js

```diff
@@ -32,7 +32,7 @@ const readFile = (files) => {
 
 const ensureUrl = (url) => {
   // emoving multiple slashes after the protocol if it exists, or after the beginning of the string otherwise
-  return url.replace(/(^\w+:|^)\/{2,}/, '$1/');
+  return url.replace(/(^\w+:|^)\/{2,}/, '$1//');
```

Comment:
Do we really need to "fix" this for the user? Any helpful fix that we do has the chance of breaking a valid (?) use-case. For example, some file url look like this: `"file:///path/to/file` ([example](https://superuser.com/questions/352133/why-do-file-urls-start-with-3-slashes)). It's a contrived example and likely won't come up in the real world, but there might be other examples...

Reply:
Yes, this is an outlier and a rare case. If this is accepted to be merged, we can raise an enhancement issue for handling this case. That is if it is even needed for bruno. Thanks for you review.